core/arch/arm/kernel/thread.c

```diff
@@ -167,24 +167,24 @@ void thread_check_canaries(void)
 	size_t n;
 
```

Comment:
For all these canaries i choose the panic way since canaries are weak but ultimated detection of a major failure.

Please tell if ones do not agree.